### DISCUSSION TITLE
Treasure Island Art

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1582] [Art]
Calling all swashbuckling treasure hunters. Rumours are spreading throughout Sunflower Land of a distant island with hidden treasure! What is this mysterious island and what does it look like?

Treasure island is a beach styled island where players can explore and dig up hidden treasure.

This is an art request for 'beach' and 'treasure' styled assets for the game.

## Inspiration

The latest Sunnyside pack includes some great beach and treasure assets which we will use as the core inspiration for the game.

![treasure_island_example](https://user-images.githubusercontent.com/11745561/198917746-129b97c9-bf1b-4e09-a181-f358dd05df95.png)

# Requests

There are a range of ways to contribute to treasure island and we encourage all types of artists to get involved ❤️ As always we lean on the community imagination and there are no limits to what you can provide. We have given a few examples of 'themed' items if you wish to use them.

## 1. Bumpkins

We will want a range of Bumpkin accessories and Bumpkin NPCs for the beach.

You can either contribute wearable items for your Bumpkin or interesting NPCs for in-game. 

Ideas:

- Pirate Hat + Costume (Male and female)
- Lifeguard Bumpkin
- Sun burnt Bumpkin (drinkable potion)

## 2. Music

Do you have some ideas that will fit the theme of a beach? We would love to hear your talent - more details please reach out to @Rofrtd 

## 3. Treasure Decorations + Rare Items

Designs for 'treasure' based items and decorations. When a user finds treasure, it will either be SFL or something they can place on their land. We are open to all types of decorations or utility based SFTs 🏆 

When designing a decoration or rare item please provide an example of it placed inside of the game in the Land Expansion area.

Examples include:

- Jewels or Statues
- Pirate Parrot
- Rare swords

## 4. Land Decorations

Do you love making the map beautiful for all of our players? Now is your chance to submit decorations that we can use across treasure island.

Examples:

- Hut/Buildings
- Treasure map
- Swimmers + Fish + Water animals
- Message in a bottle

# Submissions

1. Fits game aesthetic and colours of the beach
2. Original pieces of work
3. Pixel sizing is 1x1px

_By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service. Your art can be used freely by the project and all community members_

When submitting art, please include the following:

1. Your farm ID (we will make sure the artists get some goodies 🏆 )
2. A minified version of your art piece
3. An example of the art used inside of the game or on the Bumpkin

# Timeline

Treasure island will slowly be introduced over the next 6-8 weeks but we will be looking to get the core assets (especially if they influence gameplay) sooner.

As always, contributors will receive an airdrop of the game item they design (if it is tokenised) and also some goodies!

**Comment by VentinRD** [comment]
Farm ID: 10350
Discord: Ventin#5150

Coral The Mermaid
Ground
![Coral The Mermaid](https://user-images.githubusercontent.com/103960383/199048668-18740368-cbe6-4176-92a7-ed84805ff0db.gif)
Water
![Coral The Mermaid Water](https://user-images.githubusercontent.com/103960383/199048753-56f14aad-21c1-49f6-a353-f2665c9990af.gif)
Water With Bounce
![Juliet Themermaid in Water 2 ](https://user-images.githubusercontent.com/103960383/200972234-61fef98b-d84c-4221-bcfc-9b871ff6d3b7.gif)

![Coral The Mermaid Grid](https://user-images.githubusercontent.com/103960383/200972293-c6159f3a-c204-4bac-a0a3-f0f2bbf95a74.gif)

**Reply by adamhannigan** [reply]
Love it! Great idea 👌

**Comment by LittleEins** [comment]
### LittleEins
Farm ID: 73795
### Stone Turtles
![treasure-island-300](https://user-images.githubusercontent.com/104184348/200868012-23e346f9-fccf-4698-86f5-5d96c991dac8.gif)
![treasure-turtle](https://user-images.githubusercontent.com/104184348/200868066-c3b0a815-0565-425d-af55-c5fe90249774.gif)
![stone-turtle](https://user-images.githubusercontent.com/104184348/200868087-247cf0ea-2e2b-4273-8148-686ca38ce26a.gif)

### Shark Costume
![shark-costume-500](https://user-images.githubusercontent.com/104184348/200182267-3eb4bf3f-9adb-4f13-bd49-ec1790fe238e.png)
![shark-costume](https://user-images.githubusercontent.com/104184348/200182454-68d6a19e-8664-4fba-80b0-eb9a8bcf2716.png)

**Reply by FarmerKuenster** [reply]
This is amazing! I want it for my bumpkin already :)

**Reply by adamhannigan** [reply]
Amazing! ❤️

**Reply by adamhannigan** [reply]
We love the shark and are planning to make this pop up when the shark fin is clicked on ;)

**Reply by adamhannigan** [reply]
<img width="508" alt="Screen Shot 2022-11-24 at 2 08 30 pm" src="https://user-images.githubusercontent.com/11745561/203685376-671685a3-6c00-4bf1-bd2d-01bfcc8ad575.png">

We have just included an Easter Egg on the main land if you click on the shark fin ;)

**Comment by Whaitte** [comment]
**Farm ID: 919**

![21-35-05-198917746-129b97c9-bf1b-4e09-a181-f358dd05df95.png](https://user-images.githubusercontent.com/104186329/200220313-9066e96e-16b6-4212-a72e-107944e71854.png)
![Untitled 11-06-2022 11-03-32 (4).png](https://user-images.githubusercontent.com/104186329/200220323-e1e53fdf-3ebb-4aa9-baec-ca14f07c82e5.png)

Description: _A lost sword, used in the war between Humans and Goblins.  how did she get here??_

**Comment by VentinRD** [comment]
Farm ID: 10350
Discord: Ventin#5150

This can be use as a treasure hunt reward NFT.

The idea is to add this Monkey layout (Lets call it monkey totem) to the treasure hunt drop, similar to the eggs and mutant chicken. Once someone collects 3 of this Monkey Totem then they can exchange it for a Three Pirate Monkeys NFT. 

Not sure how the treasure hunt is going to work, but the boost can be something like 15% more SFL from treasure hunt or a higher chance of getting treasure.

Monkey Totem
![Moneky Totem 2](https://user-images.githubusercontent.com/103960383/200724082-ef3d6900-bedb-47f0-8a3a-19af0a0ed71d.png)

![Moneky Totem Grid 2](https://user-images.githubusercontent.com/103960383/200724145-04e563b7-40e8-481b-9cef-6e2d42647d80.png)

Three Pirate Monkeys
![Three Pirate Monkeys 2](https://user-images.githubusercontent.com/103960383/203722094-ececc723-9555-4d35-91e5-0ccf3f3adbf0.gif)

![Three Pirate Monkeys Grid2](https://user-images.githubusercontent.com/103960383/203723541-5494f3aa-02bd-4bf7-a8ef-c0467504e276.gif)

**Reply by adamhannigan** [reply]
These characters look very cheeky!

Is it possible that the middle one could also be bobbing?

**Reply by VentinRD** [reply]
Just updated  it. Let me know if it looks better 
Thank u Adam.

**Comment by dubuminho** [comment]
**Lifeguard Outfit**

![lifeguard](https://user-images.githubusercontent.com/107245706/200756685-535786ad-0a25-4795-9d01-ea1e59093bcf.png)

Hat:
![LGhat](https://user-images.githubusercontent.com/107245706/200756764-50a97ece-97cf-4fc3-86cf-3f16df3e5ebc.png)

Shirt:
![LGshirt](https://user-images.githubusercontent.com/107245706/200756795-2244a766-abbc-47b1-a5ca-cc9db24a1978.png)

Shorts:
![LGshort](https://user-images.githubusercontent.com/107245706/200756854-4635779c-bee4-49ba-8b4a-6564eae88084.png)

Sandals:
![LGsandals](https://user-images.githubusercontent.com/107245706/200756899-06229107-e9df-4482-80dc-fc83359df63e.png)

Lifebuoy:
![LGlifebuoy](https://user-images.githubusercontent.com/107245706/200756954-6225ce67-25e3-4010-a3d6-82261c1f3372.png)

Ps, shout out to shinon and her amazing beach bg 😁

`Farm ID: 82122`

**Reply by adamhannigan** [reply]
This fits the background perfectly! ⭐ Great job

**Reply by dubuminho** [reply]
Happy to know that! thank you adam 💗

**Reply by adamhannigan** [reply]
Could you please include this also on the new Bumpkin Github discussion ❤️ This is helping us organise all the submissions.

https://github.com/sunflower-land/sunflower-land/discussions/1748

Do you also have an idea for what the NPC character would look like? I'd love to add this into the game 🚀

**Reply by dubuminho** [reply]
Github didn't notified me this comment 🥲

Anyway, just submitted it there including the npc version (just a static version of it 😆)

**Comment by joaojuniorpt** [comment]
_**JuninkiLL
ID farm: 138257
ID discord: juninkill#1620**_


_made with love_
![ilhajuninkill02](https://user-images.githubusercontent.com/20995533/201487909-49ebf12c-a142-46eb-8d32-79369469859f.gif)

![ilhajuninkill](https://user-images.githubusercontent.com/20995533/201488087-1159b735-0e4e-4d44-825d-ab7643b0934a.gif)

Map
![mapadotesouro](https://user-images.githubusercontent.com/20995533/201488121-bd97a096-30c0-4029-9049-8977575069f3.gif)
Barrel
![barril01](https://user-images.githubusercontent.com/20995533/201488138-02033768-f539-4eff-91e0-b75c6460d55c.gif)
Gold 
![ouro](https://user-images.githubusercontent.com/20995533/201488167-4668b40a-4df1-4e0e-8fd8-da9bb036c024.gif)

**Comment by On1el** [comment]
Just as an idea
![ПАЛЬМА](https://user-images.githubusercontent.com/104461938/201488991-858b3480-a1d3-4e26-891c-c064a28f8af3.gif)

**Comment by sfwhat** [comment]
### Skull Treasure Chest 

![treasureisland-export](https://user-images.githubusercontent.com/95937808/202426041-99df2e27-35ff-4e2e-9109-538bf7cb437f.png)


With shadow ![SKULLCHEST](https://user-images.githubusercontent.com/95937808/202425952-986120ab-a254-41be-9aea-e24bdb91c7d3.png)
Without ![SKULLCHESTno shadow](https://user-images.githubusercontent.com/95937808/202425957-64c91716-d445-4feb-b26f-28a906d7baaa.png)
![SKULLCHESTBIG](https://user-images.githubusercontent.com/95937808/202425962-8aa480b1-33e7-46a1-bf95-8060e2dc574a.png)

**Reply by adamhannigan** [reply]
This looks like a quest related item for sure! 😮

**Comment by esp123** [comment]
Captain Hook inspired Pirate Bumpkin
*argh*
![pirate1](https://user-images.githubusercontent.com/74337946/204715486-2bd0a45d-9cc9-4ab2-944f-808425af6d65.png)
![pirate-parrot](https://user-images.githubusercontent.com/74337946/205176962-23161346-3fbf-4082-960e-bbd56d40e309.png)
![pirate-legs](https://user-images.githubusercontent.com/74337946/205176993-7265dbef-d101-4f22-97e6-b4bdf96d05f5.png)
![brit-pirate1](https://user-images.githubusercontent.com/74337946/205487753-6702af73-993a-4e8f-9c64-640e7a43b6ef.png)
Hat
![pirate-hat](https://user-images.githubusercontent.com/74337946/204715478-4b838ea6-87e5-44cc-99a2-d5eb17296421.png)
British Hair also known as *Rolled Towels*
![brit-hair](https://user-images.githubusercontent.com/74337946/205487745-55be443b-645e-4192-af33-1cfaafa62cec.png)
Face
![eyepatchL](https://user-images.githubusercontent.com/74337946/205487731-9862da6f-c380-43ea-a8cc-5d98d4b294d4.png)
![pirate-eyeL](https://user-images.githubusercontent.com/74337946/204715464-2faba5d7-2778-4e6c-8051-28a11ce30d77.png)
![eyepatchR](https://user-images.githubusercontent.com/74337946/205487737-bdf2d409-f2a0-4bc8-872a-a0ac3aa3f3b5.png)
![pirate-eyeR](https://user-images.githubusercontent.com/74337946/204715470-172ba313-ce50-4edc-9b38-e2bd8e0a3428.png)
Hand
![pirate-hook](https://user-images.githubusercontent.com/74337946/204715481-7215d5df-9b9c-42b8-9c6e-b28e2371d080.png)
Parrot
![parrot](https://user-images.githubusercontent.com/74337946/205176810-8d825e03-4f64-47fa-bc94-b3e0889bf62c.png)
![parrot1](https://user-images.githubusercontent.com/74337946/205176818-4404d229-9e43-4883-94b9-e09284b99996.png)
Tops
![pirate-top](https://user-images.githubusercontent.com/74337946/205176873-145352dc-09c2-4b1a-81cf-6a91e166ea2e.png)
Bottoms
![pirate-bot](https://user-images.githubusercontent.com/74337946/204715491-951f135a-aa27-4796-bcfb-8b79a8af56d8.png)
Wooden Legs by On1el
![wooden legs](https://user-images.githubusercontent.com/74337946/205177057-4259e079-e015-4cd0-938a-5575f6a5e948.png)
Shoes
![feather-shoes](https://user-images.githubusercontent.com/74337946/204716676-01749d04-4ed5-4ac6-ab86-104a4617acb0.png)

Discord: deeefault#0033

**Comment by stevew00dy** [comment]
Love this - looks amazing. We will also need a Pirate Background :)

**Comment by stevew00dy** [comment]
Could someone also possibly create a treasure map and also a piece of a treasure map.

**Comment by ghost** [comment]
**Treasure Maps**

Preview: 
![image](https://user-images.githubusercontent.com/114235416/208536882-e84892c6-33a3-434c-bd68-5dcfd1e8f801.png)

Minified: 
![image](https://user-images.githubusercontent.com/114235416/208537185-606d3a9b-778d-4886-ae83-952ce5ca36c7.png)
![image](https://user-images.githubusercontent.com/114235416/208537210-b8863990-2881-43c5-9dc8-ae05f4ada890.png)

Farm ID: 58821

**Comment by Whaitte** [comment]
**Pirate Background**
_V1_
![Pirate ship (4).png](https://user-images.githubusercontent.com/104186329/215272354-2c45260d-1102-420b-8bc2-930f4a530263.png)
Background:
![Pirate ship (5).png](https://user-images.githubusercontent.com/104186329/215272385-f1f79d98-1263-4f82-bc50-e6b61a0eb8ec.png)
pirate sword:
![sword 2.png](https://user-images.githubusercontent.com/104186329/208776985-0c558b89-21fd-441f-acec-97bcf206920b.png)
_V2_
![beige_examplke (5).png](https://user-images.githubusercontent.com/104186329/215272579-b09e3fe6-81cd-4720-a6da-5602266845b0.png)
Background:
![pirate ship (6).png](https://user-images.githubusercontent.com/104186329/215272597-0b57752b-5ac2-4bc6-b58b-5810df2a33fa.png)
pirate sword:
![sword 1.png](https://user-images.githubusercontent.com/104186329/208777181-80137e2c-a916-4ae8-89ff-ec34197694de.png)

**Straw hat**
![pirate.png](https://user-images.githubusercontent.com/104186329/208798110-5189f418-7539-405c-af59-27223e48ebf4.png)
![straw hat .png](https://user-images.githubusercontent.com/104186329/208798127-a73cb316-9ec0-4afb-b300-d9b1fc1e7bd7.png)

Farm ID: 919
Discord: whaitte#3465

**Reply by Telknub** [reply]
I want this straw hat 🥺

**Comment by archila79** [comment]
Hello, I would like to know, please, your opinion about these ideas that I am working on, any contribution is welcome to continue developing my art, thanks:
![playa](https://user-images.githubusercontent.com/115169880/209696321-cf2d2418-80b5-4b4f-8a50-ea4ad3fd0bb3.jpg)
![pixil-frame-0 (3)](https://user-images.githubusercontent.com/115169880/209696323-d9802a17-8e02-4103-ae29-ffab7b2f8e66.png)
![barco](https://user-images.githubusercontent.com/115169880/209696325-cb8c8854-4a0b-4bda-94d0-d623cb23883f.jpg)
![pixil-frame-0](https://user-images.githubusercontent.com/115169880/209696326-187e457b-c33e-4ba1-8614-64b0cb8b9951.png)
![pixil-frame-0 (13)](https://user-images.githubusercontent.com/115169880/209696374-d9ee8b0e-c9cb-40ca-a51c-86b30a826f91.png)
![pixil-frame-0 (12)](https://user-images.githubusercontent.com/115169880/209696376-78f99ded-f380-4b2f-8510-0ed6ade2802d.png)
![pixil-frame-0 (11)](https://user-images.githubusercontent.com/115169880/209696377-7b374cdb-febd-4940-9609-793f5a732159.png)
![pixil-frame-0 (10)](https://user-images.githubusercontent.com/115169880/209696378-54b29369-5992-4343-afc0-03580beb563b.png)
![pixil-frame-0 (9)](https://user-images.githubusercontent.com/115169880/209696379-8426c743-70e2-4b4b-b123-c6b61038ad2c.png)
![pixil-frame-0 (8)](https://user-images.githubusercontent.com/115169880/209696380-89cede25-6162-4ca9-9c1c-fa7309488b21.png)
![pixil-frame-0 (7)](https://user-images.githubusercontent.com/115169880/209696383-c2299d5f-8bad-42fc-a931-11b3f6b08aba.png)
![pixil-frame-0 (6)](https://user-images.githubusercontent.com/115169880/209696386-d5329679-c658-41c3-ba22-e0b3c4e2a4b6.png)
![pixil-frame-0 (5)](https://user-images.githubusercontent.com/115169880/209696387-e33c0003-a0c0-47f0-894f-f11fbe70d3c1.png)
![pixil-frame-0 (2)](https://user-images.githubusercontent.com/115169880/209696389-7731bbfb-de0f-4a77-a2e6-abb1b24a1d65.png)
![pixil-frame-0 (1)](https://user-images.githubusercontent.com/115169880/209696391-860136cd-7aad-424e-baf9-f89126142103.png)
![pixil-frame-0 (4)](https://user-images.githubusercontent.com/115169880/209696392-af6fe63f-af36-41a8-8472-6281510c9c0e.png)
![pixil-frame-0 (15)](https://user-images.githubusercontent.com/115169880/209696393-51e7fe42-41bb-44d9-92ff-51d791f00dec.png)
![pixil-frame-0 (14)](https://user-images.githubusercontent.com/115169880/209696395-10ce470b-bf7b-4e5c-877f-b2818a6b610f.png)
 

ID FARM: #143991

**Reply by stevew00dy** [reply]
I think these look incredible. Would love to see them in the game.

**Comment by Whaitte** [comment]
![pearl.png](https://user-images.githubusercontent.com/104186329/210875726-f4a99021-f5ab-4cf2-8220-8ac5b1e0dd34.png)
![pearl (1).png](https://user-images.githubusercontent.com/104186329/210875785-a6105183-4310-4964-9cbd-ce2b49c48ebc.png)
![pearl (2).png](https://user-images.githubusercontent.com/104186329/210875799-f62c81d1-a138-4b3f-9a62-1b0c7bee7b2b.png)


**Pearl necklace **
_V1_
![Pearl necklace .png](https://user-images.githubusercontent.com/104186329/210875829-d9767c22-7426-4549-be75-7740ef74343b.png)
_V2_
![Pearl necklace  (1).png](https://user-images.githubusercontent.com/104186329/210875859-1881ebaf-000a-4414-9d19-561c2a9ab2fa.png)

**scepter of pearls**
_V1_
![pearl scepter (4).png](https://user-images.githubusercontent.com/104186329/210876018-faeae097-5774-441f-a607-2f4ca677a097.png)![pearl scepter (3).png](https://user-images.githubusercontent.com/104186329/210876023-6f9fad62-f42f-4332-a53b-67649d5f4b68.png)
_V2_
![pearl scepter (2).png](https://user-images.githubusercontent.com/104186329/210876518-ab68b891-2fb1-4be7-8287-c188ff0ea7ad.png)![pearl scepter (1).png](https://user-images.githubusercontent.com/104186329/210876530-85f96eeb-b109-4d9c-8575-f03003a2e150.png)
_V3_
![pearl scepter.png](https://user-images.githubusercontent.com/104186329/210876483-a93c3213-5399-413b-9bdc-260be98a3d12.png)![beige_examplke (1).png](https://user-images.githubusercontent.com/104186329/210876493-16dad1fb-92f1-4a8f-aabe-2460394149c1.png)

Farm ID: 919
Discord: whaitte#3465

**Comment by digital-farms** [comment]
`land id: 21637`
`discord: KOTOB#4391`

**PEARL**
![pearl_example_v2](https://user-images.githubusercontent.com/105886943/212567772-ef0b5068-94cc-4b54-95df-74d450dd5c27.png)

**Minified:** ![pearl_v2](https://user-images.githubusercontent.com/105886943/212567773-a90f4ffb-a3f4-4752-928d-6ee96d3f5576.png)

V2
![pearl_example](https://user-images.githubusercontent.com/105886943/212536812-d4acb0c4-e441-4701-b798-2dac01db6fcf.png)

**Minified:** ![pearl](https://user-images.githubusercontent.com/105886943/212536813-1050f7d0-0493-479a-8f09-879f95702254.png)

**Comment by Galphath** [comment]
Hi I saw on the proposals that some stuff is missing like the pearl, I hope you like it

**Farm** : 170
**Discord** : La abuela

**PEARL**

Light up

![image](https://user-images.githubusercontent.com/20891096/212589773-cf9dd77b-c3b8-47af-b4c2-8419b91877d3.png)

![pearl alter2](https://user-images.githubusercontent.com/20891096/212589793-62b210a6-82ce-457a-8c03-b0a424a889b3.png)

![image](https://user-images.githubusercontent.com/20891096/212589830-f72f86f9-8125-4347-b21b-c55f611a700a.png)

![pearl alter1](https://user-images.githubusercontent.com/20891096/212589841-03d645a9-495a-4db8-80d1-76f5a544dd74.png)

Light down

![image](https://user-images.githubusercontent.com/20891096/212578400-da2bcd9a-622e-437e-b164-20e92fec0b68.png)

![pearl](https://user-images.githubusercontent.com/20891096/212501577-1bbd105d-9d62-4bc0-8bbd-6a0a65a52314.png)

![image](https://user-images.githubusercontent.com/20891096/212578821-c0087f13-e410-429a-aa9f-7829802c5709.png)

![pearl2](https://user-images.githubusercontent.com/20891096/212578837-4ef34d2b-8710-44ea-9ae8-558883b73f88.png)


**PIPI**
![pipi_sample](https://user-images.githubusercontent.com/20891096/212520264-d94a6ce3-a2ab-4f74-a248-a39827a9dde9.png)

1
![pipi1](https://user-images.githubusercontent.com/20891096/212502731-c6678934-a30f-4928-81e3-024e6a3451db.png)
2
![pipi2](https://user-images.githubusercontent.com/20891096/212502732-a387c7a3-dec1-454a-8d90-cb515bea6145.png)

no pearl 
![image](https://user-images.githubusercontent.com/20891096/212577741-99785603-f0c8-440c-9fbc-5948cb23ce1f.png)

3
![pipi3](https://user-images.githubusercontent.com/20891096/212578007-a71b34fa-739f-4e13-9630-a95bd73b5bf6.png)

4
![pipi4](https://user-images.githubusercontent.com/20891096/212578042-f4c33786-a3cc-410b-910f-ec8c2e416ac1.png)


**PIRATE CAKE**
![cake sample](https://user-images.githubusercontent.com/20891096/212520279-b49f6867-7d11-45a2-9c41-44477ce447ef.png)

![pirate_cake](https://user-images.githubusercontent.com/20891096/212520282-48ea3cb8-7ab0-49ca-ab68-c979f88257b5.png)

**SEAWEED**

![image](https://user-images.githubusercontent.com/20891096/212792629-ecdfeb9c-7a6c-4b44-9038-43892eb771c9.png)

![seaweed](https://user-images.githubusercontent.com/20891096/212792717-0da77540-c337-4316-9fae-d68dc9670bec.png)

**Comment by esp123** [comment]
Me Monkee Hat
v1
![monkee-hat1](https://user-images.githubusercontent.com/74337946/212814284-466f3cb1-0ee5-4b29-ae2e-9619db38ad8f.png)
![monkee hat1](https://user-images.githubusercontent.com/74337946/212814324-4b589c66-960a-4edc-b3f5-8407f0926fe4.png)

v2
![monkee-hat2](https://user-images.githubusercontent.com/74337946/212814341-b76a9bb4-1c89-484f-92de-f7b323f22bdc.png)
![monkee hat2](https://user-images.githubusercontent.com/74337946/212814349-7eda5b08-8fda-4f06-842e-f99ffb4686cf.png)

v3
![monkee-hat3](https://user-images.githubusercontent.com/74337946/212814458-ebb8efb9-5e31-462f-9a21-4a9276c48086.png)
![monkee hat3](https://user-images.githubusercontent.com/74337946/212814480-a5eaf265-861c-4306-92db-4101b09e600f.png)

v4
![monkee-hat5](https://user-images.githubusercontent.com/74337946/213171801-04277be2-592c-4063-adff-1799cd8abd72.png)
![monkee hat5](https://user-images.githubusercontent.com/74337946/213171796-c3076395-f305-49be-8589-94b52879c195.png)

Made improvements from madam birb's suggestions (no more dithering/lesser)

suggestions are welcome
discord: deeefault#0033

**Reply by Life-Is-Great** [reply]
🔥

**Comment by Galphath** [comment]
**BUMPKINS**  didn't wanted to stretch the other post.

**PENDING** :  

· NPCs


Farm : 170
Discord : La Abuela



**SAMPLES**
![beach samples 400percent](https://user-images.githubusercontent.com/20891096/212814602-1cafaec1-46dc-4470-9bb6-cfd3198e6cbe.png)
![sample1](https://user-images.githubusercontent.com/20891096/212815137-0686ffa3-2b0f-4faa-8abb-e893cc4bc4d3.png)
![Sample2](https://user-images.githubusercontent.com/20891096/212815139-74a0ebda-910c-45cb-8a4c-cf5a52972c2d.png)

**HAT AND HAIRSTYLES**

![Sunhat](https://user-images.githubusercontent.com/20891096/213030785-14156fc9-6687-455a-a754-eb8a8fa50ebf.png)

![SFL2077_hair](https://user-images.githubusercontent.com/20891096/212814088-c2172386-03c6-4f95-a229-82de39d05098.png)
![Sunflare_hair](https://user-images.githubusercontent.com/20891096/212814090-905f2a08-d632-4ad1-9028-cea42bb35df1.png)
![BlackHoleSun_hair](https://user-images.githubusercontent.com/20891096/212815159-4b541884-9bc3-4897-b9f4-8e2a92bb8cfc.png)


**SWIMGEAR**

![Latexswimsuit](https://user-images.githubusercontent.com/20891096/212814276-fae78604-48cc-4583-bc25-997489fa1923.png)
![swimsuit_lines](https://user-images.githubusercontent.com/20891096/212814280-e87b93e4-00ed-4572-878e-c83e1c2c6619.png)
![Bikini](https://user-images.githubusercontent.com/20891096/212814283-3324b832-56bd-42f0-95e1-b5395de4d298.png)

Sarong with texture 

![Sarong_blue](https://user-images.githubusercontent.com/20891096/212814277-a1f5a640-2d45-4dfd-96b8-9e8c3985e433.png)
![Sarong_orange](https://user-images.githubusercontent.com/20891096/212814279-6737666d-0527-48ac-84d3-ca14c25a0db7.png)

Sarong without texture 

![Sarong_orange_noText](https://user-images.githubusercontent.com/20891096/213843648-75e2ec7d-1d66-4881-8949-b053e8981de4.png)
![Sarong_blue_noText](https://user-images.githubusercontent.com/20891096/213843649-ce6abbc2-396b-4a11-b9c2-994508fe4a8f.png)


**SHOES**

![Chancla](https://user-images.githubusercontent.com/20891096/212814325-0215f3a5-19fa-4948-bc53-22379d5fa10c.png)
![sandals](https://user-images.githubusercontent.com/20891096/212814326-e4cf23f6-b0e5-46d3-af4b-f3a3f5ac4980.png)

**HANDHELD**

![Martini](https://user-images.githubusercontent.com/20891096/212814372-44718b93-258b-4c0f-a661-88318f6321f7.png)
![Surfboard](https://user-images.githubusercontent.com/20891096/212814375-ee92420c-ed13-4083-9719-a2d206ae1b8b.png)

**BACKGROUNDS**

![Background](https://user-images.githubusercontent.com/20891096/212814432-9825b623-3fb2-43de-aea4-f77b549b89ef.png)
![BackgroundTowel](https://user-images.githubusercontent.com/20891096/212814434-ca62f8b0-7118-415c-be63-39450a7da46a.png)



-------------------------

**FILES**

[BeachFemaleAssets.zip](https://github.com/sunflower-land/sunflower-land/files/10440026/BeachFemaleAssets.zip)

**Reply by Life-Is-Great** [reply]
🔥

**Reply by On1el** [reply]
Her hat is really something

**Comment by Life-Is-Great** [comment]
### **1) Fitness Bumpkin**
Modern bumpkin liking the beach life. All dressed up for the beach =D 
![All parts2-export](https://user-images.githubusercontent.com/108883042/199700268-3794e46a-8165-4f68-8f76-3da9a4aed25c.gif)
![Bumpkin](https://user-images.githubusercontent.com/108883042/199702031-2ae07e76-7f27-4699-b3aa-dc43ee7b4982.png) ![Hat](https://user-images.githubusercontent.com/108883042/199702059-a41c67a2-8901-43c6-beae-5bd6e2b97583.png) ![Hat2](https://user-images.githubusercontent.com/108883042/199702065-8300fc8f-d36c-475d-bf8a-e3e613423500.png) ![ItemDrink](https://user-images.githubusercontent.com/108883042/199702088-bc9d921b-207d-4040-8c0d-3f18ea833ca2.png) ![ItemCane](https://user-images.githubusercontent.com/108883042/199702096-46e0b8e2-b4d7-4925-8563-57b277f984e6.png) ![Necklase](https://user-images.githubusercontent.com/108883042/199702106-72100f22-f2a9-4a3d-ad71-fce4ad80f9b5.png) ![Shirt-png](https://user-images.githubusercontent.com/108883042/199702122-99319806-5c06-459b-afab-680a6dff1350.png) ![Shoes](https://user-images.githubusercontent.com/108883042/199702139-3b072693-f56c-4e09-9bd0-85a9f84670bd.png) ![Shorts](https://user-images.githubusercontent.com/108883042/199702170-0169dc71-4846-4bc7-b7b0-2af9c67b209a.png) ![Background-Disco](https://user-images.githubusercontent.com/108883042/199702198-19fb1212-0ecb-4bbc-ba6b-7f87b712e23f.png) ![Background-Beach](https://user-images.githubusercontent.com/108883042/199702224-c92a89dc-d66b-423a-83e6-e690c8601e59.png)




### 2) Those would not be bad as a rewards (Both for treasure island and globally)
Farming Cake | Tools Cake
--- | ---
![Farming Cake-export](https://user-images.githubusercontent.com/108883042/199696519-f6c5effc-297d-472a-8b24-7c3483cd7dac.png) ![Farming Cake](https://user-images.githubusercontent.com/108883042/199696141-06f95776-fde5-4c58-863b-1b7456471a6a.png)  | ![Tools Cake-export](https://user-images.githubusercontent.com/108883042/199696354-83fa4f30-0808-4f5a-a32c-3d23d22bb874.png) ![Tools Cake](https://user-images.githubusercontent.com/108883042/199699613-e7b5f555-e2ee-4873-9e86-f14649341b98.png)
